andrewinsoul/TravelPortal
# setup model

**What does this PR do?**
It sets up the model

**Description of Task to be completed?**
- add models
- created corresponding migration file

**How should this be manually tested?**
- Fetch branch locally by running `git fetch origin setup-model`
- Activate virtual environment
- Ensure a .env file is in your root directory looking like the .env.sample file
- Run `python manage.py migrate`
- The tables city and flight should be added to your database
